### PR TITLE
Added Coding da Vinci

### PR DESCRIPTION
**Pull request by Dumbaz**
German Hackaton starting end of this month

Reference Issue : #1 #4

### CODE REVIEW

File: data/hacks.json [hunk 1]
```diff
@@ -38,5 +38,13 @@
     "location" : "Santa Clara, CA, USA",
     "website"  : "http://wilhacks.com/",
     "logo"     : "https://challengepost-s3-challengepost.netdna-ssl.com/photos/production/challenge_photos/000/529/752/datas/full_width.png"
+  },
+  {
+    "name" : "Coding Da Vinci",
+    "start" : "21/10/2017",
+    "end" : "2/12/2017",
```
Comment (geekyd) [inline data/hacks.json]: @Dumbaz Thanks for the PR can you please make it `02/12/2017`

Comment (Dumbaz) [reply data/hacks.json]: Sure, give me a sec. Also, this is of course in reference to #1, but I assume you´d also take additions to conferences.json?

Comment (geekyd) [reply data/hacks.json]: @Dumbaz Yes please have a look at #4 & #5

**Comment by geekyd** [comment]
@Dumbaz Thanks for the PR :)

**Pull request merged**